composer.json

```diff
@@ -20,7 +20,7 @@
     }
   },
   "require": {
-    "php": ">=8.0",
+    "php": ">=8.2",
```

Comment:
I saw we don't have any version on this lib. Meaning the customers/integrators that require this lib in master branch can experience PHP version error during composer update. This can be seen as a BC break.
Is it what we want?

Adding a description in the PR would help to understand the context :wink:

Reply:
Sorry we have tags, I looked at the wrong place. Ok then